lambdas/adobe-sign-api/serverless.yml

```
@@ -28,6 +28,22 @@ functions:
             parameters:
               paths:
                 id: true
+  createSigningUrlAgreement:
+    role: lambdaRole
+    handler: lib/agreements-controller.createSigningUrlAgreement
+    vpc:
+      securityGroupIds:
+        - { 'Fn::ImportValue': 'AdobeSign-ServerlessSecurityGroup-GroupId' }
+      subnetIds:
+        - { 'Fn::ImportValue': 'AdobeSign-PrivateSubnetA-Ref' }
+    events:
+      - http:
+          path: agreements/{template_id}
```

I think maybe we change this to:

```suggestion
          path: templates/{template_id}/create
```

Or

```suggestion
          path: agreements/create/{template_id}
```

---

If you take the first, I already have the templates controller that you can put it in. We are creating an agreement though... Your cal!